indico_themes_canonical/static/css/ubuntu-summit/surveys.css

```diff
@@ -13,10 +13,10 @@
 
 .i-form fieldset legend {
   font-size: 1.306rem;
+  font-weight: 300;
   line-height: 2rem;
```

Comment:
You don't have to do this here now, but I guess something worth exploring.

These 3 properties keep repeating in multiple places. Would it be possible to refactor them into some basic styles and reuse them with @extend or some other native function?

Reply:
There is probably a better way to handle a lot of the repeated styles. The issue is the level of specificity used in the original styles are difficult to overwrite. If we were using SCSS it would be possible, definitely something to consider for next time